### PR TITLE
feat: tools add Dash

### PR DESCRIPTION
**Pull request by P13LIAM**

### CODE REVIEW

File: docs/docs/09-tools.md [hunk 1]
```diff
@@ -39,6 +39,13 @@ https://azure.microsoft.com/services/storage/blobs
 
 Azure Blob Storage เป็นพื้นที่จัดเก็บไฟล์บนคลาวด์ของทาง Microsoft Azure ที่เหมาะสำหรับ Unstructured data โดยเฉพาะ
 
+## Dash
+
+https://dash.plotly.com/  
+https://github.com/plotly/dash
```
Comment (kahnwong) [inline docs/docs/09-tools.md:44]: I think we could make the header a hyperlink

Comment (P13LIAM) [reply docs/docs/09-tools.md:44]: like this?

```
## [Dash](https://dash.plotly.com/)
```

Comment (kahnwong) [reply docs/docs/09-tools.md:44]: ✅

**Review by kahnwong — APPROVED** [review]
🎉

**Review by zkan — APPROVED** [review]
Looking good!

**Pull request merged**